imap_processing/codice/codice_l1a.py

```diff
@@ -625,7 +700,9 @@ def group_ialirt_data(packets: xr.Dataset, data_field_range: range) -> list[byte
     return grouped_data
 
 
-def create_binned_dataset(apid: int, dataset: xr.Dataset) -> xr.Dataset:
+def create_binned_dataset(
+    apid: int, dataset: xr.Dataset, science_values: list[str]
```

Comment:
Now passing in `science_values` as a parameter to make it easier to re-use this function in multiple places.